helpers/constants.js

```diff
@@ -2,6 +2,7 @@ import { soliditySha3 } from "web3-utils";
 import { BN } from "bn.js";
 import shortid from "shortid";
 
+const X = new BN(0).notn(256); // Shorthand for UINT256_MAX for domain proofs
```

Comment:
I like it shorter since visually, when scanning vertically it's less obtrusive since the lines using `X` are still lined up with the lines using other domains. But I suppose `MAX` could be a decent compromise -- much more clear while still being relatively succinct.

Reply:
Ok we'll do `UINT256_MAX`